### ISSUE TITLE
cifar10_main data format inconsistency?

### ISSUE THREAD

**Issue by h8907283** [tensorflow/models#3795]
Hi @nealwu.

The following is from the _tower_fn function in cifar10_main.py in cifar10_estimator ( https://github.com/tensorflow/models/blob/master/tutorials/image/cifar10_estimator/cifar10_main.py):

  model = cifar10_model.ResNetCifar10(
      num_layers,
      batch_norm_decay=batch_norm_decay,
      batch_norm_epsilon=batch_norm_epsilon,
      is_training=is_training,
      data_format=data_format)
  logits = model.forward_pass(feature, input_data_format='channels_last')

If the input parameter 'data_format' is 'channel_first', then why do we apply it to ResNetCifar10, but not to forward_pass? Why does forward_pass get a hard-coded data format?

Thanks for your pointer. I'm still getting up to speed on this one.

**Comment by nealwu** [comment]
I believe what this means is that the desired `data_format` is `'channels_first'`, but the input data is given in the format `'channels_last'`, which means it needs to be transposed. See in particular: https://github.com/tensorflow/models/blob/master/tutorials/image/cifar10_estimator/cifar10_model.py#L45
